### ISSUE TITLE
Remove main.css

### ISSUE THREAD

**Issue by ghost** [Chemaclass/knob-base#6]
Hello,
I found the file that is causing the malfunction of the control panel. The file is public/css/main.css.

Commenting on the line 55 in file vendor/chemaclass/src/lib/Actions.php everything back to function properly.

Alternatively remove file main.css in public/css

**Comment by Chemaclass** [comment]
Yeah, you're right. I already moved  from knob-base to the theme template (knob-mvc), because this Actions should be a decision from the template implementation itself, not from the kernel :)

**Comment by ghost** [comment]
Not work again. The main.css file is always loaded in the control panel and causes corruption.

![admin](https://cloud.githubusercontent.com/assets/14064365/23023391/ab722472-f455-11e6-924d-9701fd55c4e4.PNG)
![source](https://cloud.githubusercontent.com/assets/14064365/23023390/ab6e523e-f455-11e6-8cb6-418d95250839.PNG)

**Comment by Chemaclass** [comment]
What I did was move the Actions from knob-base to knob-mvc. My idea of Knob-mvc was to be the basic structure where start to develop having the power of knob-base. 

You are allow to change the Actions in your theme, as far as the theme is your own implementation. Same apply with the stylish of your blog/web/app.

I didn't focus on styling, while I was developing this framework, 'cos I wanted to care first on the real WP structure, and how could we handle easier in order to simulate a MVC framework. 
We could say that:
* Knob-mvc: focus on styling and personalized web/app using the kernel: Knob-base.
* Knob-base: shouldn't be focus on any style of the page, but deal with WP, and provide models instead, in order to get rid of "ugly WP styling code" [personal opinion ;)] 

I need to update the `Readme` of both projects including that, that's true. 

I think if this is a suggestion regarding the styling, in that case, feel free to open an Issue in Knob-mvc, so we will discuss there. But, for sure, if the code-problem is from WP, kernel structure or something that should be inside the Knob-base, feel free to open here an Issue, then we could work on it.
